scarlet/initialisation.py

```diff
@@ -0,0 +1,136 @@
+from .source import *
+from .interpolation import interpolate_observation
+from .observation import Observation
+from .wavelet import Starlet
+
+def build_initialisation_coadd(observations, wavelet = False):
```

Comment:
American English, please!

Reply:
But I do think it's a good idea to bundle initialization methods here, to clean up `sources.py`

Reply:
+1

Reply:
Kept and americaniSed.